rfc9204.md

```diff
@@ -1395,7 +1378,7 @@ immediately sent due to flow control is not affected by this limit.
 Implementations should limit the size of unsent data, especially on the decoder
 stream where flexibility to choose what to send is limited.  Possible responses
 to an excess of unsent data might include limiting the ability of the peer to
-open new streams, reading only from the encoder stream, or closing the
+open new streams, reading only from the encoder stream or closing the
```

Comment:
```suggestion
open new streams, reading only from the encoder stream, or closing the
```
This is a list of three possible mitigations; I believe the Oxford comma is appropriate.